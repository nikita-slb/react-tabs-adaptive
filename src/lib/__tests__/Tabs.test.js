/* eslint getter-return: "off" */

import React from 'react';
import { Tabs } from '../index';
import renderer from 'react-test-renderer';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('Tabs', () => {
  it('renders properly', () => {
    const tree = renderer
        .create(
                <Tabs/>
            )
            .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
