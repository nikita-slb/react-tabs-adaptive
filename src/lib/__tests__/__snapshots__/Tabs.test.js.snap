// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs renders properly 1`] = `
<div
  className="Tabs"
>
  <div
    className="Tabs-Items-container"
  >
    <div
      className="Tabs-Items-wrap"
    >
      <div
        style={
          Object {
            "height": "100%",
            "overflow": "hidden",
            "position": "relative",
            "width": "100%",
          }
        }
      >
        <div
          style={
            Object {
              "WebkitOverflowScrolling": "touch",
              "bottom": 0,
              "left": 0,
              "marginBottom": 0,
              "marginRight": 0,
              "overflow": "scroll",
              "position": "absolute",
              "right": 0,
              "top": 0,
            }
          }
        >
          <ul
            className="Tabs-Items"
          />
        </div>
        <div
          style={
            Object {
              "display": "none",
              "height": 6,
              "position": "absolute",
            }
          }
        >
          <div
            style={
              Object {
                "backgroundColor": "rgba(0,0,0,.2)",
                "borderRadius": "inherit",
                "cursor": "pointer",
                "display": "block",
                "height": "100%",
                "position": "relative",
              }
            }
          />
        </div>
        <div
          style={
            Object {
              "borderRadius": 3,
              "bottom": 2,
              "display": "none",
              "position": "absolute",
              "right": 2,
              "top": 2,
              "width": 6,
            }
          }
        >
          <div
            style={
              Object {
                "backgroundColor": "rgba(0,0,0,.2)",
                "borderRadius": "inherit",
                "cursor": "pointer",
                "display": "block",
                "position": "relative",
                "width": "100%",
              }
            }
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="Tabs-Active-Content"
  />
</div>
`;
